### DISCUSSION TITLE
Problem with entering COM frequency.

### DISCUSSION THREAD

**Discussion by WrongwayUK** [nguyenquyhy/Flight-Tracker-StreamDeck#177] [Q&A] [answered]
Just installed and started using Flight Tracker with profiles provided by DARK FORDY on YouTube. Using the NAV & COM function, it seems to read a key ok however I am unable to enter a number in the third decimal place using the keypad. An example frequency is EGNH (Blackpool) with an ATIS frequency of 127.205 I cannot enter the last 5
Also it doesn't seem to change frequencies in the A320 although it does seem to work with the XPDR. Just wondering if I am doing something wrong.

**Comment by rmroc451** [comment, marked as answer]
This is the simconnect inputs only allowing us to pass 4 digits, and the sim displaying something else.

This is documented on one of the open issues, and needs attention from the sdk devs.

I have a zendesk ticket out on these discrepancies. No eta on a resolution yet.  For now, you just have to use the atc menu to tune those freqs that are wonky.

I will like that issue, when I'm near a pc. On mobile, atm.

**Reply by WrongwayUK** [reply]
Understood and thanks for the prompt reply. I can save some of my hair now. :)
